src/components/sidepanel/SidePanelCSS.js

```diff
@@ -0,0 +1,26 @@
+import { colors } from "../colors/index";
+
+export const SidePanelCSS = `
+
+width: 139px;
+height: 26px;
+
+font-family: 'HK Grotesk', sans-serif;
+font-style: normal;
+font-weight: 700;
+font-size: 1.6rem;
+line-height: 26px;
+
+letter-spacing: -0.45px;
+
+
+color: ${colors.white};
+
+&:hover {
+    background: ${colors.midnight500};
+    align-items: center;
+    padding: 18px 66px 18px 66px;
```

Comment:
move padding to outside the &:hover. it's causing the text to jump around. also remove position:absolute
![sidepanel](https://user-images.githubusercontent.com/70789275/194683805-404ce9df-5d97-4c3d-9455-99605e1125e7.gif)

Reply:
A transition for the changing background colour and setting the cursor to pointer (`cursor: pointer;`) upon hover would be cool

Reply:
gotcha! btw how do u guys see it like that? cause when i run it i don't see the black background thing like you guys do?

Reply:
@samanthatuhseng if you open chrome dev tools you can inject styles into HTML elements manually. i added `background: black` in the root element so that I could see the white text